inc/saiport.h

```diff
@@ -1955,6 +1955,16 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_DUAL_MEDIA,
 
+    /**
+     * @brief IPG is mandated by IEEE 802.3 to ensure reliable communication between two ethernet devices.
+     * The minimum recommended inter-packet gap(IPG) is 96 bits (9.6usec for 10GBASE-T and 0.96usec for 100GBASE-T)
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 96 (bits)
```

Comment:
remove "bits" string, only raw number is supported

Reply:
removed